django/db/models/fields/__init__.py

```diff
@@ -977,6 +983,12 @@ def to_python(self, value):
                 params={'value': value},
             )
 
+    def rel_db_type(self, connection):
+        """
```

Comment:
It doesn't seem that docstrings are duplicated on other subclass overridden methods -- okay with you to remove it?